test/ArcxAnalyticsSdk.test.ts

```diff
@@ -449,6 +458,25 @@ describe('(unit) ArcxAnalyticsSdk', () => {
       })
     })
 
+    describe('#_trackClicks', () => {
+      it('does not track clicks', () => {
+        const eventStub = sinon.stub(analyticsSdk, 'event')
+
+        window.dispatchEvent(new window.Event('click'))
+
+        expect(eventStub).to.not.have.been.called
+      })
+
+      it('catches click event', () => {
+        const eventStub = sinon.stub(analyticsSdk, 'event')
+
+        analyticsSdk['_trackClicks']()
+        window.dispatchEvent(new window.Event('click'))
+
+        expect(eventStub).calledOnceWith(CLICK_EVENT, { elementId: undefined, content: undefined })
+      })
```

Comment:
This test is not very useful as is. We could leave it like that but then we need to add tests in `ArcxAnalyticsProvider.test.tsx` with proper catching of `elementId`

Reply:
I'd tried on friday, but something went wrong with tests with this functionality. I'll try to figure it out for another hour